testsuite/tests/kuadrant/authorino/operator/clusterwide/conftest.py

```diff
@@ -40,4 +50,3 @@ def commit(request, commit, authorization2):
     """Commits all important stuff before tests"""
     request.addfinalizer(authorization2.delete)
     authorization2.commit()
-    authorization2.wait_for_ready()
```

Comment:
This should now pass and should be included

Reply:
Won't pass on wildcard test, but probably can be overwritten to wait for hostsNotLinked